data-client/src/main/java/org/wikipedia/csrf/CsrfTokenClient.java

```diff
@@ -210,4 +214,30 @@ public void twoFactorPrompt() {
     private interface RetryCallback {
         void retry();
     }
+
+    /**
+     * Class CsrfTokenCallExecutor which implement callable interface to get CsrfTokenCall.
+     */
+    class CsrfTokenCallExecutor implements Callable<Response<MwQueryResponse>> {
+        private Service service;
```

Comment:
javadoc for this member, for instance say in 5 to 10 words what this service offers.

Reply:
Done.